### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | yes
| Fixed tickets | N/A
| License       | MIT

### CODE REVIEW

**Comment by j0k3r** [comment]
Maybe that should be a good PR to add Lighttpd configuration too :) 
:arrow_right: https://github.com/wallabag/wallabag/issues/1931

**Comment by nicosomb** [comment]
Documentation visible here http://doc.wallabag.org/en/update-documentation/index.html

File: docs/en/user/installation.rst [hunk 1]
```diff
@@ -151,22 +188,58 @@ Assuming you install wallabag in the /var/www/wallabag folder, here's the recipe
 
 After reloading or restarting nginx, you should now be able to access wallabag at http://domain.tld.
 
-.. note::
+.. tip::
 
     When you want to import large file into wallabag, you need to add this line in your nginx configuration ``client_max_body_size XM; # allows file uploads up to X megabytes``.
 
+Configuration on LigHTTPd
```
Comment (j0k3r) [inline docs/en/user/installation.rst]: It's lighttpd in lowercase :)

File: docs/en/user/installation.rst [hunk 2]
```diff
@@ -151,22 +188,58 @@ Assuming you install wallabag in the /var/www/wallabag folder, here's the recipe
...
+
+::
+
+    server.modules = (
+    "mod_fastcgi",
+    "mod_access",
+    "mod_alias",
+    "mod_compress",
+    "mod_redirect",
+    "mod_rewrite",
```
Comment (j0k3r) [inline docs/en/user/installation.rst]: You should add one level of indentation here, same for `url.rewrite-if-not-file` below

File: docs/en/user/installation.rst [hunk 3]
```diff
@@ -151,22 +188,58 @@ Assuming you install wallabag in the /var/www/wallabag folder, here's the recipe
...
+    compress.filetype = ( "application/javascript", "text/css", "text/html", "text/plain" )
+    include_shell "/usr/share/lighttpd/use-ipv6.pl " + server.port
+    include_shell "/usr/share/lighttpd/create-mime.assign.pl"
+    include_shell "/usr/share/lighttpd/include-conf-enabled.pl"
+    dir-listing.activate = "disable"
+
+    url.rewrite-if-not-file = (
+    "^/([^?])(?:\?(.))?" => "/app.php?$1&$2",
+    "^/([^?]*)" => "/app.php?=$1",
+    "^/wiki$" => "/app.php",
```
Comment (j0k3r) [inline docs/en/user/installation.rst]: I think you can remove that line, we don't have route starting with /wiki

File: docs/en/user/installation.rst [hunk 4]
```diff
@@ -151,22 +188,58 @@ Assuming you install wallabag in the /var/www/wallabag folder, here's the recipe
...
 
-.. note::
+.. tip::
 
     When you want to import large file into wallabag, you need to add this line in your nginx configuration ``client_max_body_size XM; # allows file uploads up to X megabytes``.
 
+Configuration on LigHTTPd
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Edit your ``lighttpd.conf`` file and paste this configuration into it:
```
Comment (j0k3r) [inline docs/en/user/installation.rst]: You should also precise that line (from Apache configuration):

> Assuming you install wallabag in the ``/var/www/wallabag`` folder, here's the recipe for wallabag :

**Pull request merged**